src/UserService.Models.Dto/Responses/Avatar/AvatarsResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using LT.DigitalOffice.UserService.Models.Dto.Models;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.UserService.Models.Dto.Responses.Avatar
+{
+  public record AvatarsResponse
```

Comment:
why is it avatar? if user has two or more image he has only one avatar, but with this name few, is it okay?